pkg/cmd/kind/delete/clusters/deleteclusters.go

```diff
@@ -48,7 +48,7 @@ func NewCommand(logger log.Logger, streams cmd.IOStreams) *cobra.Command {
 		},
 	}
 	cmd.Flags().StringVar(&flags.Kubeconfig, "kubeconfig", "", "sets kubeconfig path instead of $KUBECONFIG or $HOME/.kube/config")
-	cmd.Flags().BoolVar(&flags.All, "all", false, "delete all clusters")
+	cmd.Flags().BoolVar(&flags.All, "all", false, "delete all nodes (containers) labeled as kind cluster and kubeconfig entries for the nodes")
```

Comment:
Not sure how necessary saying `nodes (containers)` is, but thought it helped a little.

Reply:
this flag still seems like the wrong place to explain what deleting does.
the command usage seems more appropriate

Reply:
(or documentation)

Reply:
Fair. I like the idea of usage, but it might be worthwhile to also have in docs as well.